src/binary/ibc-setup/commands/connect.ts

```diff
@@ -0,0 +1,103 @@
+import fs from 'fs';
+import path from 'path';
+
+import yaml from 'js-yaml';
+
+import { Link } from '../../../lib/link';
+import { appFile, registryFile } from '../../constants';
+import { AppConfig } from '../types';
+import { loadAndValidateApp } from '../utils/load-and-validate-app';
+import { loadAndValidateRegistry } from '../utils/load-and-validate-registry';
+import { resolveRequiredOption } from '../utils/options/resolve-required-option';
+import { resolveHomeOption } from '../utils/options/shared/resolve-home-option';
+import { resolveKeyFileOption } from '../utils/options/shared/resolve-key-file-option';
+import { resolveMnemonicOption } from '../utils/options/shared/resolve-mnemonic-option';
+
+import { createClient } from './ics20';
+
+export type Flags = {
+  readonly interactive: boolean;
+  readonly home?: string;
+  readonly keyFile?: string;
+  readonly mnemonic?: string;
+  readonly src?: string;
+  readonly dest?: string;
+};
+
+export type Options = {
+  readonly home: string;
+  readonly mnemonic: string;
+  readonly src: string;
+  readonly dest: string;
+};
+
+export async function connect(flags: Flags) {
+  const home = resolveHomeOption({ homeFlag: flags.home });
+  const app = loadAndValidateApp(home);
+  if (!app) {
+    throw new Error(`${appFile} not found at ${home}`);
+  }
```

Comment:
This is probably not right, related: https://github.com/confio/ts-relayer/pull/95/files#r590706164